doc/source/examples/devportal.rst

```diff
@@ -0,0 +1,41 @@
+.. _ref_devportal_examples:
+
+
+Developer portal articles
+=========================
+
+In this section, you'll find informative and insightful blog
```

Comment:
Is this section going to feature dev portal postings on PyMAPDL? The tone there is informal. In the PyAnsys documentation, we avoid "our" and "we"--and focus on "you" (the developer/user). My proposed changes reflect this.

Reply:
Good call. My idea is to link PyMAPDL Dev portal articles.. I thought the "pymapdl" was implicit.

I can change that to `PyMAPDL Developer Portal articles`